curriculum/challenges/english/16-the-odin-project/top-learn-html-foundations/html-foundations-question-a.md

```diff
@@ -15,7 +15,8 @@ Closing tags tell the browser where an element ends. They are almost the same as
 
 A full paragraph element looks like this:
 
-<img src="https://cdn.statically.io/gh/TheOdinProject/curriculum/90b1a362af0bb8635af9593cd8911c9aefb68569/foundations/html_css/html-foundations/imgs/00.png" alt="element diagram">
+<img src="https://cdn.freecodecamp.org/curriculum/odin-project/html-foundations/html-foundations-01.png" alt="GPT
+The image shows a basic HTML paragraph element. It consists of an opening tag <p>, followed by the words 'some text content' representing the content of the paragraph, and concludes with a closing tag </p>. There are labels pointing to each part: 'Opening Tag' at the beginning, 'Content' in the middle, and 'Closing Tag' at the end.">
```

Comment:
Just a little too much detail. Don't need the information about arrows pointing to things. Just describe the HTML in the image.

"An opening p tag, followed by the content string "some text content", followed by a closing p tag."